static/js/protein/data_processing_utils.ts

```diff
@@ -287,6 +287,34 @@ export function getVarGeneAssoc(data: GraphNodes): ProteinVarType[] {
               }
               if (n2.nodes[0].value) {
                 tissue = n2.nodes[0].value;
+        */
+        if (n.property !== "geneSymbol") {
+          continue;
+        }
+        for (const n1 of n.nodes) {
+          for (const n2 of n1.neighbors) {
+            if (n1.neighbors.length !== 4) {
+              continue;
+            }
+            console.log(n1.neighbors);
+            if (n2.property === "log2AllelicFoldChange") {
+              if (_.isEmpty(n2.nodes[0].value)) {
```

Comment:
curious why here you're checking that the value is not empty & in the rest of the code you're checking that the value is not null? is there a reason for the difference in pattern?

Reply:
actually null and empty are not the same, isEmpty catches more things as documented here: https://lodash.com/docs/4.17.15#isEmpty